src/store/ducks/plans.ts

```diff
@@ -577,68 +577,73 @@ export const getReason = (_: Registry, props: PlanFilters) => props.reason;
  * Gets an array of Plan objects filtered by jurisdictionIds
  * @param {Registry} state - the redux store
  * @param {PlanFilters} props - the plan filters object
  */
-export const getPlansArrayByJurisdictionIds = createSelector(
-  [plansArrayBaseSelector, getJurisdictionIds],
-  (plans, jurisdictionIds) =>
-    jurisdictionIds
-      ? plans.filter(plan =>
-          jurisdictionIds.length ? jurisdictionIds.includes(plan.jurisdiction_id) : true
-        )
-      : plans
-);
+export const getPlansArrayByJurisdictionIds = (planFilter?: string) =>
+  createSelector(
+    [plansArrayBaseSelector(planFilter), getJurisdictionIds],
+    (plans, jurisdictionIds) =>
+      jurisdictionIds
+        ? plans.filter(plan =>
+            jurisdictionIds.length ? jurisdictionIds.includes(plan.jurisdiction_id) : true
+          )
+        : plans
+  );
 
 /** getPlansArrayByStatus
  * Gets an array of Plan objects filtered by plan status
  * @param {Registry} state - the redux store
  * @param {PlanFilters} props - the plan filters object
  */
-export const getPlansArrayByStatus = createSelector(
-  [plansArrayBaseSelector, getStatusList],
-  (plans, statusList) =>
-    statusList
-      ? plans.filter(plan => (statusList.length ? statusList.includes(plan.plan_status) : true))
-      : plans
-);
+export const getPlansArrayByStatus = (planFilter?: string) =>
+  createSelector(
+    [plansArrayBaseSelector(planFilter), getStatusList],
+    (plans, statusList) =>
+      statusList
+        ? plans.filter(plan => (statusList.length ? statusList.includes(plan.plan_status) : true))
+        : plans
+  );
 
 /** getPlansArrayByReason
  * Gets an array of Plan objects filtered by FI plan reason
  * @param {Registry} state - the redux store
  * @param {PlanFilters} props - the plan filters object
  */
-export const getPlansArrayByReason = createSelector(
-  [plansArrayBaseSelector, getReason],
-  (plans, reason) => (reason ? plans.filter(plan => plan.plan_fi_reason === reason) : plans)
-);
+export const getPlansArrayByReason = (planFilter?: string) =>
+  createSelector(
+    [plansArrayBaseSelector(planFilter), getReason],
+    (plans, reason) => (reason ? plans.filter(plan => plan.plan_fi_reason === reason) : plans)
+  );
 
 /** getPlansArrayByParentJurisdictionId
  * Gets an array of Plan objects filtered by plan jurisdiction parent_id
  * @param {Registry} state - the redux store
  * @param {PlanFilters} props - the plan filters object
  */
-export const getPlansArrayByParentJurisdictionId = createSelector(
-  [plansArrayBaseSelector, getParentJurisdictionId],
-  (plans, parentJurisdictionId) =>
-    plans.filter(
-      plan =>
-        (parentJurisdictionId && !plan.jurisdiction_path ? false : true) &&
-        (parentJurisdictionId && plan.jurisdiction_path
-          ? plan.jurisdiction_path.includes(parentJurisdictionId)
-          : true)
-    )
-);
+export const getPlansArrayByParentJurisdictionId = (planFilter?: string) =>
```

Comment:
Nitpick: Feeling iffy  calling this planFilters, reason being there is an interface `PlanFilters` in the same file that describes a different type of variable.

Reply:
@p-netm Changes updated